### DISCUSSION TITLE
关于streamlit run main.py的问题

### DISCUSSION THREAD

**Discussion by Pandoravirus-N-T** [zai-org/ChatGLM3#1094] [Q&A]
运行streamlit run main.py命令显示
Usage: streamlit run [OPTIONS] TARGET [ARGS]...
Try 'streamlit run --help' for help.
Error: Invalid value: File does not exist: main.py
但我都打开main文件了，想请教一下问题出哪里了

**Comment by lukeewin** [comment]
not found this file, you should change directory to ChatGLM3/composite_demo

**Comment by Pandoravirus-N-T** [comment]
你好作者，我尝试了您说的方法后出现了如下报错
Traceback (most recent call last):
  File "D:\anaconda\Scripts\streamlit-script.py", line 6, in <module>
    from streamlit.cli import main
ModuleNotFoundError: No module named 'streamlit.cli'
但我检查了Streamlit包的位置已经在对应环境下，能请教一下啊问题出哪里了嘛，麻烦作者了
